gradle/dependencies.gradle

```diff
@@ -173,6 +173,7 @@ libs += [
   metrics: "com.yammer.metrics:metrics-core:$versions.metrics",
   mockitoCore: "org.mockito:mockito-core:$versions.mockito",
   mockitoInline: "org.mockito:mockito-inline:$versions.mockito",
+  mockitoJunitJupiter: "org.mockito:mockito-junit-jupiter:$versions.mockito",
```

Comment:
Is this library necessary to do the test? I'm asking because this will make our build have more dependency, not sure if it's required.

Reply:
This library supports `MockitoExtension`. 
We can not use `@RunWith(MockitoJUnitRunner.class)` in junit5, we need to use `@ExtendWith(MockitoExtension.class)` instead.

Reply:
OK, but what I want to ask is, could we do the test without `mockitoJunitJupiter`? If yes, why should we use this library since we didn't use it before. Thanks.

Reply:
Maybe I can replace `Mockito.mock`  and `Mockito.spy` with `@Mock` and `@Spy` annotation.
Then we don't need  `mockitoJunitJupiter` library.
But it will have a `unchecked` warning.

Reply:
@chia7712 , what do you think about this?

Reply:
As we want to replace PowerMock/easymock by mockito, it is useful to introduce that library to simplify the migration. Especially, there are many tests using similar PowerMock/easymock annotation.

In short, it should be fine to me :)

Reply:
Thanks, @chia7712 , I have not more comments. :)